### ISSUE TITLE
Add cache busting option to dev server

### ISSUE THREAD

**Issue by kuon** [gohugoio/hugo#3897]
When doing theme development, many files are are cached by the browser and I have to "hard reload" to get latests resources.

I propose to add an option to `hugo server --noHttpCache` that would add the following header to every requests:

```
Cache-Control: no-store, no-cache, must-revalidate, max-age=0
Pragma: no-cache
```

To be used only in development of course.

**Comment by bep** [comment]
Chrome has an option in their dev console to turn off cache.

**Comment by kuon** [comment]
Yes I know, but it aggressively turn it on again, and I also test with mobile browsers. The only way I found to clear the cache on some mobile browsers is to change the port. If this feature is not added I'll have to reverse proxy the hugo server to add the headers, but I think this should be built in.

**Comment by bep** [comment]
@kuon I agree that this is a good idea, I just wanted to give you a temporary workaround if you didn't already know. 

Pull requests are welcome!

**Comment by kuon** [comment]
woaw that was fast:) great! 

On September 22, 2017 5:14:18 PM GMT+02:00, "Bjørn Erik Pedersen" <notifications@github.com> wrote:
>Closed #3897 via 0b34af216154367af7f53ce93d44e6b3d58c3f34.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/gohugoio/hugo/issues/3897#event-1261435488

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
